packages/spark-core/settings/_settings.scss

```diff
@@ -555,10 +526,6 @@ $narrow-nav-left-border: 5px solid $red !default;
 $narrow-nav-link-font-weight: normal !default;
 
 //
-// Pagination
-///
-$pag-breakpoint: 24.25rem !default;
```

Comment:
How does this keep getting back in the codebase haha. I removed it with the var table add last release. 🤷🏻‍♂️